### ISSUE TITLE
Socialization upgrade

### ISSUE THREAD

**Issue by Beumac** [Daenks/Meridian59_103#304]
It's really sad that in the world of Meridian 59 that the majority of it's building players feel forced into not guilding for a variety of reasons.

Unable to build safely
Can't work spells in towns without fear etc

Now some people may argue that this is carebear and people should just build guilded. Without trying to argue that point either way. People almost ALWAYS elect to just not guild while they build. This leaves a large number of quiet noobs that are left in darkness. They simply don't socialize because it's harder for them. 

This is just a bad idea to leave as is in my eyes.

My suggested proposal to boost socialization is to make the lowest rank in a guild no different than being solo with the added benefit that they can at least socialize with people freely and easily.

I think this is a very neutral change that every side could agree on.

This change should allow no other benefits to lowest ranked guild members other than socializing and being able to carry on as they would un guilded. This change would also make it necessary to enforce that players that have been in pvp combat would not be allowed to switch back to lowest rank for some time. So that people inclined to abuse it would not be able to do so. I would also make it so that, lowest ranked guild members do not get the bio stamp of the guild as this just attracts unwanted contact that they might as well be not guilded anyways.

Let's bring the community back together! No more waiting in the dark, opting to not make friends for your own safety.

TL;DR Make lowest ranked guild members have the benefits of an un-guilded player to promote socializing and a better play experience.

/discuss!

**Comment by dalu** [comment]
Or you could just not pick fights and not join guilds that are lead by aggressive douches?

**Comment by SinM59** [comment]
Actually last few servers we attempted to play on, we were targeted and
griefed. We play for pvp and don't need to be aggressive douches against
those who aren't the big boys on the server also looking for pvp.

**Comment by dalu** [comment]
Are you talking about the random pks? Those are part of the game. They add suspense to a boring build process.
There are send groups for a reason. To keep your guild secret until you can afford to show yourself.
However, the whole idea of guild wars not coloring people is just a bad idea.
How ever you try to regulate something the more room for abuse it brings and then you add measures to counter those measures to counter more measures.
Simple rules, no expections.
If anything this "guild wars don't color people's names" should be removed because that's just a free ticket for griefing.
And the necromancer game should be re-enabled so people have a reason to stay white.

There is no "just a little pregnant" either you're playing a pvp enabled game or you don't.
Personally I've been on the good side and the bad side. Red life isn't easy. You're hated by most and you need resources to fuel your game. When you die you lose a lot. Chances for revenants are high and fighting them off is annoying and you're exposed to ambushes while you are fighting them.
White life is nice. You're on the good side, people love you and you're even getting gifts occasionally. At least I did. Personally I prefer white life.

But then again the game has changed and I don't play anymore anyway, except once every 1-2 years when I get nostalgia
